packages/blocks/src/api/test/registration.js

```diff
@@ -1528,6 +1528,74 @@ describe( 'blocks', () => {
 			);
 		} );
 
+		// Check the `usesContext` array is correct.
+		it( 'should reject invalid usesContext property', () => {
+			registerBlockBindingsSource( {
+				name: 'core/testing',
+				label: 'testing',
+				usesContext: 'should be an array',
+			} );
+			expect( console ).toHaveWarnedWith(
+				'Block bindings source usesContext must be an array.'
+			);
+		} );
+
+		it( 'should add usesContext when only defined in the client', () => {
+			// Simulate server bootstrap.
+			registerBlockBindingsSource( {
+				name: 'core/testing',
+				label: 'testing',
+				usesContext: [ 'postId', 'postType' ],
+			} );
```

Comment:
I see you're using `addBootstrappedBlockBindingsSource()` to simulate the server behavior in [a related PR](https://github.com/WordPress/gutenberg/pull/63797/files#). Should we be taking that approach in this PR as well?

Reply:
That makes sense 🙂 I'll make that change.

Reply:
I made the change as part of [this commit](https://github.com/WordPress/gutenberg/pull/63513/commits/1d7160491a65c402a9b2ca5da6d4c1a87ee0b8c7).

Reply:
Nice, this improves the readability of the test 👍🏻